containers/Ideas/Idea.js

```diff
@@ -312,6 +326,7 @@ export const Idea = (props) => {
         <ScrollView
           style={styles.container}
           scrollEnabled={!isEditing}
+          keyboardShouldPersistTaps='handled'
```

Comment:
needed because otherwise the scrollview will swallow the tap if the keyboard is open